main.go

```diff
@@ -75,9 +76,17 @@ func main() {
 		log.Fatal(err)
 	}
 
+  methods := make(map[string]resize.InterpolationFunction)
+  methods["nearest"] = resize.NearestNeighbor
+  methods["bicubic"] = resize.Bicubic
+  methods["bilinear"] = resize.Bilinear
+  methods["mitchell"] = resize.MitchellNetravali
+  methods["lanczos2"] = resize.Lanczos2
+  methods["lanczos3"] = resize.Lanczos3
```

Comment:
You can initialise this as a constant using the map literal syntax like this:

```
const methods = map[string]resize.IterpolationFunction(
  "nearest": resize.NearestNeighbor,
  "bicubic": resize.Bicubic,
  // ...
)
```

You might also want to do it outside the `main()` function, rather than inline like this.